src/CodeStyle/Core/CodeFixes/Host/Mef/CodeStyleHostLanguageServices.MefHostExportProvider.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Composition.Hosting;
+using System.Linq;
+using System.Reflection;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.Host
+{
+    internal sealed partial class CodeStyleHostLanguageServices : HostLanguageServices
```

Comment:
MEF based detection of languages services in CodeStyle layer. https://github.com/dotnet/roslyn/issues/41462 tracks alternative approaches once https://github.com/dotnet/roslyn/pull/41363 is merged in.

Reply:
👍